comparecompanies.py

```diff
@@ -1,9 +1,11 @@
 import sys
 import numpy as np
-
+import parser as pa
 def main(argv):
 	print "Hello World"
 	# TODO Xin: Call parsing methods here. Get us to the point where we have a data structure that contains all the companies and all the links.
-
+pa.links('https://en.wikipedia.org/wiki/List_of_S&P_500_companies')
```

Comment:
Call this within the main function and assign it to something.
`links = pa.links()`